### PR TITLE
[einthusan] adding .com tld

### PR DESCRIPTION
**Pull request by tlonic**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

einthusan.tv is having some issues with their normal domain, and have added einthusan.com as an additional domain. My addition extracts the tld and downloads from that tld, specifically.

### CODE REVIEW

File: youtube_dl/extractor/einthusan.py [hunk 1]
```diff
@@ -18,7 +19,7 @@
 
 
 class EinthusanIE(InfoExtractor):
-    _VALID_URL = r'https?://einthusan\.tv/movie/watch/(?P<id>[^/?#&]+)'
+    _VALID_URL = r'https?://einthusan\.(?P<tld>tv|com)/movie/watch/(?P<id>[^/?#&]+)'
```
Comment (dstftw) [inline youtube_dl/extractor/einthusan.py]: Capture entire host with scheme.

Comment (tlonic) [reply youtube_dl/extractor/einthusan.py]: done

File: youtube_dl/extractor/einthusan.py [hunk 2]
```diff
@@ -18,7 +19,7 @@
 
 
 class EinthusanIE(InfoExtractor):
-    _VALID_URL = r'https?://einthusan\.tv/movie/watch/(?P<id>[^/?#&]+)'
+    _VALID_URL = r'https?://(?P<host>einthusan\.(tv|com))/movie/watch/(?P<id>[^/?#&]+)'
```
Comment (dstftw) [inline youtube_dl/extractor/einthusan.py]: Do not capture groups you don't use.

Comment (dstftw) [inline youtube_dl/extractor/einthusan.py]: Add test.

**Pull request merged**